Makefile

```diff
@@ -75,35 +73,35 @@ run: generate fmt vet lint
 deploy: manifests
 	# Remove the certmanager certificate if KSERVE_ENABLE_SELF_SIGNED_CA is not false
 	cd config/default && if [ ${KSERVE_ENABLE_SELF_SIGNED_CA} != false ]; then \
-	${KUSTOMIZE} edit remove resource ../certmanager; \
-	else ${KUSTOMIZE} edit add resource ../certmanager; fi;
-	${KUSTOMIZE} build config/default | kubectl apply -f -
+	kubectl delete --ignore-not-found=true -k ../certmanager; \
```

Comment:
Will it delete the resource definition from kustomize file or from the cluster ?

Reply:
From the cluster

Reply:
@sivanantha321 Would you like to help test the changes in this PR to see if it works for your setup?

Reply:
This is not working. We have to remove the [certmanager resources from the kustomize file](https://github.com/kserve/kserve/blob/32bc9d69b60434ed33dd4639d7255a418f445327/config/default/kustomization.yaml#L16) if KSERVE_ENABLE_SELF_SIGNED_CA is true.

Reply:
I have tested all other targets. It is working fine.

Reply:
Could you try again?

Reply:
Looks like `kubectl kustomize` does not have sub-commands like `edit`. I thought I could do something like `kubectl edit remove resource -k ../certmanager;` but unfortunately that's not available.

Reply:
I updated the script to remove the content in `certificate.yaml` as a workaround. Let me know if you have better alternatives.

Reply:
I don't have any other ideas either.